runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerConfig.java

```diff
@@ -44,4 +45,15 @@ default Color markerColor()
 	{
 		return Color.YELLOW;
 	}
+
+	@ConfigItem(
+			position = 1,
+			keyName = "clearLocalMarkers",
+			name = "Clear Local Marks",
```

Comment:
Not sure on the usage of `Local` here, might be worth using the same terminology as the client `Region`